docker-compose.yml

```diff
@@ -90,7 +90,6 @@ services:
       - "6061:6061"
     security_opt:
       - no-new-privileges:true
-      - read_only:true
```

Comment:
Is this `read_only: true` breaking the appliance?
Ideally, this should work by mounting a dir in a volume/tmpfs with `rw` and `noexec` privs.

Reply:
yes, unfortunately this particular docker-compose service container is performing a build and this read_only filesystem is breaking it.  FWIW this is only used for CI/local development.